api/core/model_runtime/model_providers/xinference/speech2text/speech2text.py

```diff
@@ -107,20 +119,8 @@ def _speech2text_invoke(
         :param temperature: The sampling temperature, between 0 and 1. Higher values like 0.8 will make the output mor            e random,while lower values like 0.2 will make it more focused and deterministic.If set to 0, the model wi            ll use log probability to automatically increase the temperature until certain thresholds are hit.
         :return: text for given audio file
         """
-        if credentials['server_url'].endswith('/'):
```

Comment:
Recommended to keep this piece of compatible code.

Reply:
OK